frontend/src/Components/Table/VirtualTable.js

```diff
@@ -83,11 +101,10 @@ class VirtualTable extends Component {
       className,
       items,
       scroller,
+      scrollTop: ignored,
```

Comment:
Instead of this we should pass `{...otherProps}` at the top instead of the bottom so other props don't override explicitly set props.